packages/protocol/contracts-0.8/governance/Validators.sol

```diff
@@ -1054,8 +1054,8 @@ contract Validators is
    * @param value New reset period for slashing multiplier.
    */
   function setSlashingMultiplierResetPeriod(uint256 value) public nonReentrant onlyOwner {
-    allowOnlyL1();
     slashingMultiplierResetPeriod = value;
+    emit SlashingMultiplierResetPeriodSet(value);
```

Comment:
Slashing multiplier is no longer used by any L2 contracts. is there a reason for keeping them active?